util/po2php.php

```diff
@@ -107,12 +109,13 @@ function escape_s($match){
 				$out .= '"'.$v.'"';
 			}
 			if ($k != "") {
-				$out .= $arr?");\n":";\n";
+				/// @TODO Maybe add parentheses here?
```

Comment:
`$arr ? 'foo' : 'bar'` is a minimized if/else block. to make it easier noticeable, i added parentheses around `$arr`.